### PR TITLE
Remove a few more Win32 memory functions.

### PR DESCRIPTION
**Pull request by audetto**
CopyMemory, MoveMemory, FillMemory.

### CODE REVIEW

File: source/Windows/WinVideo.cpp [hunk 1]
```diff
@@ -257,7 +257,7 @@ void VideoBenchmark () {
   // WITH FULL EMULATION OF THE CPU, JOYSTICK, AND DISK HAPPENING AT
   // THE SAME TIME
   DWORD realisticfps = 0;
-  FillMemory(mem+0x2000,0x2000,0xAA);
+  FillMemory(mem+0x2000,0xAA,0x2000);
```
Comment (tomcw) [inline source/Windows/WinVideo.cpp:260]: Change to memset()

File: source/Windows/WinVideo.cpp [hunk 2]
```diff
@@ -274,9 +274,9 @@ void VideoBenchmark () {
 	  }
     }
     if (cycle & 1)
-      FillMemory(mem+0x2000,0x2000,0xAA);
+      FillMemory(mem+0x2000,0xAA,0x2000);
```
Comment (tomcw) [inline source/Windows/WinVideo.cpp:277]: Change to memset()

File: source/Windows/WinVideo.cpp (hunk 1 above)
Comment (audetto) [reply source/Windows/WinVideo.cpp:260]: ooops

**Comment by tomcw** [comment]
Merged it.

**Pull request merged**